.gitignore

```diff
@@ -0,0 +1,62 @@
+/.c9/
```

Comment:
can we update this file to include only the files that are present in the repo? Right now it's referring to a lot of files that are not even relevant